Feat/aut974_2 Added options to SmartPeakCLI (#429)
AutoFlowResearch/SmartPeak: docs/guide/cli.rst

        ./SmartPeakCLI `
            --load-session C:/data/GCMS_SIM_Unknowns/sequence.csv `
            --report featuredb `
            --reports-out-dir $HOME/data/reports `
            --output-features $HOME/data/output_features

    .. code-tab:: bash

        ./SmartPeakCLI \
            --load-session $HOME/data/GCMS_SIM_Unknowns/sequence.csv \
            --report featuredb \

>             --reports-out-dir $HOME/data/reports `

Reviewer: Nice choice on the working and the default directory names in my opinion.

    This parameter is optional. The default value is ''.

    -l    --load-session  (required)

>     The path to session file, or a directory with structured file organization (prefer session file)

Reviewer: ```suggestion
    The path to a session file, or to a directory with a structured file organization (session file is preferred)
```

    -r    --report
    Specify a report type to export.
    This parameter is optional. The default value is '[ FeatureDB PivotTable ]'.

    -rt   --report-sample-types

>     A list of sample types to include in the report. The full list can be found in documentation https://smartpeak.readthedocs.io

Reviewer: ```suggestion
    A list of sample types to include in the report. The full list can be found in the documentation https://smartpeak.readthedocs.io
```

    This parameter is optional. The default value is '[ ALL ]'.

    -rm   --report-metadata

>     A list of metadata to include in the report. The full list can be found in documentation https://smartpeak.readthedocs.io

Reviewer: ```suggestion
    A list of metadata to include in the report. The full list can be found in the documentation https://smartpeak.readthedocs.io
```

    This parameter is optional. The default value is '[ ALL ]'.

    -ro   --reports-out-dir

>     An absolute or relative path to an output directory for reports. Overrides the default location which is the current working directory. SmartPeak will create given directory if one does not exist.

Reviewer: ```suggestion
    An absolute or relative path to an output directory for reports. Overrides the default location which is the current working directory. SmartPeak will create the given directory if one does not exist.
```

    By default the console output is colored, this flag disables colors.
    This parameter is optional. The default value is '0'.

    -pg   --disable-progressbar

>     Progress bar allows to track the progress of the entire workflow. This option disables the progress bar.

Reviewer: ```suggestion
    Progress bar allows tracks the progress of the entire workflow. This option disables the progress bar.
```

    This parameter is optional. The default value is '0'.

    -ld   --log-dir

>     The path to the log directory. Given directory has to exist. Overrides the default location for the log file: https://smartpeak.readthedocs.io/en/latest/guide/guistart.html#logs

Reviewer: ```suggestion
    The path to the log directory. The given directory has to exist. Overrides the default location for the log file: https://smartpeak.readthedocs.io/en/latest/guide/guistart.html#logs
```

    This parameter is optional. The default value is ''.

    -o    --output-features

>     An absolute or relative path to an output directory. Overrides the default location which is the current working directory. SmartPeak will create given directory if one does not exist.

Reviewer: ```suggestion
    An absolute or relative path to an output directory. Overrides the default location which is the current working directory. SmartPeak will create the given directory if one does not exist.
```
